modules/virt-creating-nad-localnet-console.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-creating-nad-localnet-console_{context}"]
+= Creating a NAD for localnet topology using the web console
+
+You can create a network attachment definition (NAD) to connect workloads to a physical network by using the {product-title} web console.
+
+.Prerequisites
+* Use nmstate to configure the localnet to OVS bridge mappings.
+
+.Procedure
+
+. Navigate to *Networking* -> *NetworkAttachmentDefinitions* in the web console.
+
+. Click *Create Network Attachment Definition*. The network attachment definition must be in the same namespace as the pod or virtual machine using it.
+
+. Enter a unique *Name* and optional *Description*.
+
+. Select *OVN Kubernetes secondary localnet network* from the *Network Type list*.
+
+. Enter the name of your pre-configured localnet identifier in the Bridge mapping field.
+
+. Optional: You can explicitly set MTU to the specified value. Defaults to the value chosen by the kernel.
```

Comment:
```suggestion
. Optional: You can explicitly set MTU to the specified value. The default value is chosen by the kernel.
```

Reply:
or "This field defaults to the value chosen by the kernel." (to avoid a sentence fragment)